Dockerfile

```diff
@@ -114,6 +119,8 @@ RUN apt-get install --no-install-recommends -y \
     ca-certificates \
     curl \
     jq \
+    libhidapi-dev \
```

Comment:
These are listed above (and another PR removes `libhidapi-dev` entirely) ... so I think they can be removed?